app/src/shared/components/Layout/Footer.jsx

```diff
@@ -17,9 +18,17 @@ const MadeBy = styled(UnstyledMadeBy)`
     }
 `
 
-const Footer = () => (
+type Props = {
+    topBorder?: boolean,
+}
+
+const FooterWithBorder = styled(FooterColumns)`
+    border-top: ${({ topBorder }) => (topBorder ? '2px' : '0')} solid #D8D8D8;
```

Comment:
2px doesn't seem right.

<img width="418" alt="Screenshot 2020-02-12 10 20 54" src="https://user-images.githubusercontent.com/320066/74320696-60e59500-4d81-11ea-9d4c-ed9325f2ea85.png">

Reply:
Ok, I was wondering about it since Abstract shows `2px`... will fix 👍